### PR TITLE
chore: Add changeset

### PR DESCRIPTION
**Pull request by gretanausedaite**
## Changes

Adding changeset to layouts repo.

## Testing

N/A

### CODE REVIEW

File: .changeset/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changesets
```
Comment (mayank99) [inline .changeset/README.md:1]: this file can be removed

File: .changeset/fair-points-cry.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (mayank99) [inline .changeset/fair-points-cry.md:1]: remove

File: .github/workflows/changeset-release.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
+      - greta/*
```
Comment (mayank99) [inline .github/workflows/changeset-release.yml:7]: remove

File: .github/workflows/changeset-release.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      - run: yarn
+
+      - name: Create release PR or publish to npm
+        uses: changesets/action@v1
+        with:
+          publish: yarn release
+          title: Release packages
+          commit: Release packages
+        env:
+          GITHUB_TOKEN: ${{ secrets.IMJS_ADMIN_GH_TOKEN }}
```
Comment (mayank99) [inline .github/workflows/changeset-release.yml:41]: i dont think this repo has IMJS_ADMIN token, so remove for now

File: .changeset/fair-points-cry.md (hunk 2 above)
Comment (gretanausedaite) [reply .changeset/fair-points-cry.md]: I was testing. removed

**Review by mayank99 — APPROVED** [review]

File: package.json [hunk 5]
```diff
@@ -29,9 +29,11 @@
     "lint": "yarn lint:js && yarn lint:css",
     "prepare": "husky install",
     "pre-commit": "lint-staged",
-    "build-docker": "yarn workspace @itwin/itwinui-layouts-demo build-docker"
+    "build-docker": "yarn workspace @itwin/itwinui-layouts-demo build-docker",
+    "release": "yarn build --filter={./packages/*}... && changeset publish"
```
Comment (mayank99) [inline package.json:33]: this won't work, we don't have turborepo in this repo

Comment (mayank99) [reply package.json:33]: this should work:
```suggestion
    "release": "yarn build && changeset publish"
```

**Pull request merged**